provider/cmd/pulumi-resource-svmkit/backend.ts

```diff
@@ -52,6 +53,24 @@ export class Backend extends pulumi.ComponentResource {
         const { connection, triggers } = args;
         const targetDir = "svmkit";
 
+        const VALIDATOR_VARIANT =
+            args.validatorConfig.validatorVariant ?? "agave";
+
+        // Note: this list must match the names defined inside solana-build.
+        const validVariants = [
+            "solana",
```

Comment:
```suggestion
```

NB: The solan validator is officially being dropped in the next 2.x.x release if we want to stop supporting as well. Just FYI.

Reply:
For sure.  When we remove it from solana-build we can remove it here.